app/cdap/components/ImportDataset/styles.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { makeStyles } from '@material-ui/styles';
+
+export const useStyles = makeStyles({
+  dropContainer: {
+    height: '400px',
+    textAlign: 'center',
+    border: '1px dashed #616161',
+    borderRadius: '4px',
+    position: 'relative',
+    zIndex: 99,
+    padding: 20,
+    width: '400px',
```

Comment:
use number instead of number with px wherever possible

Reply:
done